.travis.yml

```diff
@@ -9,8 +9,9 @@ script:
 - cd ${TRAVIS_BUILD_DIR}
 env:
   matrix:
-  - IMG=base-alpine/0.2.0
-  - IMG=base-debian/0.2.0
```

Comment:
It looks a bit strange that we need to specify versions every time and then delete them.

Reply:
Especially given that we're leaving original 0.2.0 images.

I think to follow more general software engineering practice we would have to change thoese `Dockerfile`s on every release and then just publish tags. At least it would make CI/CD simpler and reviewers wouldn't have to run `diff` locally.